src/OAuth1/Provider/Twitter.php

```diff
@@ -60,6 +60,7 @@ public function getIdentity(AccessTokenInterface $accessToken)
         $hydrator = new ArrayHydrator([
             'id' => 'id',
             'name' => 'fullname',
+            'email' => 'email',
```

Comment:
Does twitter provide email now?

Reply:
well at least for me checking the box "Request email address from users" from the app permissions tab, and adding both terms and privacy links allow me to get user email address. think you have to regenerate the token first

Reply:
If twitter provides an `email` field that is verified, we should add

```php
$user->emailVerified = true;
```

to mark it as a verified, can you check this moment?

Thanks!

Reply:
I honestly dont know if those provided emails are verified from Twitter end, as they dont provide  any indication except for mobile numbers as (bool) phone_need_verification, so i did not want to alter what was previously provided by twitter class, which might be a breaking change for someone depending on $user->emailVerified being true

Reply:
ok, let's make it false for now, i will check this moment it in the feature